content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -218,18 +218,20 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: standard
-provisioner: kubernetes.io/gce-pd
-parameters:
-  type: pd-standard
+provisioner: your-provisioner
 volumeBindingMode: WaitForFirstConsumer
 allowedTopologies:
 - matchLabelExpressions:
   - key: topology.kubernetes.io/zone
     values:
-    - us-central-1a
-    - us-central-1b
+    - zone1
+    - zone2
 ```
 
+- `provisioner`: See [`Provisioner`](/docs/concepts/storage/storage-classes/#provisioner) for details. 
+- `allowedTopologies`: defines the constraints for the allowed topologies. In this example, it's based on zone.
```

Comment:
Suggestions:

nit: Capitalize the word, Define (to be consistent with the previous sentence)
nit: Capitalize the word, Specify label expressions to enable a node ...

Reply:
I agree, I will implement this change.